app/controllers/proofs_controller.rb

```diff
@@ -4,7 +4,7 @@ def create
     form = ProofForm.new(resource_params)
     if form.save
       flash[:success] = 'Proof was successfully created.'
-      render status: 201, js: "window.location = '#{identity_path(resource_params[:identity_id])}'"
+      render status: 201, js: "window.location = '#{resource_params[:entity_id]}'"
```

Comment:
looks invalid

Reply:
It works normally, but we can use `render status: 201, js: 'location.reload(true)'`